src/client.ts

```diff
@@ -52,14 +54,22 @@ export default class UnleashClient extends EventEmitter {
     }
 
     isEnabled(name: string, context: any, fallbackValue?: boolean): boolean {
+        const newContext: Context = {
+            ...context,
+            toggleName: name,
+        };
+        return this._isEnabled(name, newContext, fallbackValue).enabled;
+    }
+
+    _isEnabled(name: string, context: Context, fallbackValue?: boolean) {
```

Comment:
can't this method the the `featureToggleDefinition` as input?

Reply:
Sure